packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -1202,7 +1202,7 @@ class _LibraryRefColumn extends TreeColumnData<Reference> {
 
 class _ClassOrInstanceCountColumn extends ColumnData<Reference> {
   _ClassOrInstanceCountColumn()
-      : super('Count', alignment: ColumnAlignment.right);
+      : super('Count', alignment: ColumnAlignment.right, fixedWidthPx: 75.0);
```

Comment:
nit: trailing comma in the modified lines in this file too

Reply:
done